ATTRIBUTION.md

```diff
@@ -585,5 +585,32 @@ of Constellation.
 <td>BSD</td>
 <td></td>
 </tr>
+<tr class="odd">
+<td>richtextfx-0.10.6.jar</td>
+<td>The 2-Clause BSD License</td>
+<td>https://opensource.org/licenses/BSD-2-Clause</td>
+<td>GNU General Public License, version 2, with the Classpath Exception</td>
+<td>http://openjdk.java.net/legal/gplv2+ce.html</td>
```

Comment:
This row has extra columns which is making the file look a little funny. If you're able to, it might be good to merge the 2 licenses into the same columns (separate by comma, break, or make it a list)